investigations/Possible_Instrument_View_Improvements.md

```diff
@@ -0,0 +1,67 @@
+# Investigation of Possible Instrument 2.0 Effects on Instrument View Performance.
+This document outlines areas in which the performance of the instrument view may benefit from the introduction of the new Instrument 2.0 caching layers. Tests were carried out using the Microsoft Visual Studio debugger and profiling tools in conjunction with `Callgrind` on Linux which provided information on which Instrument 1.0 methods are most heavily used when interacting with the instrument view. The instrumentation was taken for the duration of the creation of the `InstrumentWidget`.
+
+## Findings
+
+The following report contains results in terms of percentage inclusive time during the loading of an empty GEM instrument in the instrument view:
+
+- 34% of time is spent doing dynamic casts in several places: 
+	- [CompAssemblyActor::CompAssemblyActor](https://github.com/mantidproject/mantid/blob/7fa758ce336c6ac8ad590c6f3476311d28ed54b2/qt/widgets/instrumentview/src/CompAssemblyActor.cpp#L52)
+	- [ParComponentFactory::create](https://github.com/mantidproject/mantid/blob/546b2eccbaf236fd217ccbd21573f88aa0272103/Framework/Geometry/src/Instrument/ParComponentFactory.cpp#L58)
+	- [ComponentActor::isNonDetector](https://github.com/mantidproject/mantid/blob/7fa758ce336c6ac8ad590c6f3476311d28ed54b2/qt/widgets/instrumentview/src/ComponentActor.cpp#L69)
+	- [Component::Component](https://github.com/mantidproject/mantid/blob/master/Framework/Geometry/src/Instrument/Component.cpp#L25)
+- 49% of time is being spent on `InstrumentActor::getInstrument()` with the following breakdown:
```

Comment:
And how much of that `49%` is spent on forwarding to `ExperimentInfo::getInstrument` and below?